picard/browser/filelookup.py

```diff
@@ -62,6 +62,18 @@ def albumLookup(self, album_id):
 
     def artistLookup(self, artist_id):
         return self._lookup('artist', artist_id)
+		
+    def trackLookup(self, track_id):
+        return self._lookup('track', track_id)
+        
+    def workLookup(self, work_id):
+        return self._lookup('work', artist_id)
+        
+    def releaseGroupLookup(self, releaseGroup_id):
+        return self._lookup('release-group', releaseGroup_id)
+        
+    def acoustLookup(self, acoust_id):
+        return self.launch('https://acoustid.org/track/' + acoust_id)
```

Comment:
Please don't hardcode the url here, add it to https://github.com/musicbrainz/picard/blob/master/picard/const/__init__.py#L54-L64
and use something like `PICARD_URLS['acoustid_track']`